apps/web/src/app/api/admin/[data]/route.ts

```diff
@@ -159,6 +159,18 @@ const clients: Clients = {
         textTemplates.getApiTextTemplateManagementTemplateDefinitions(),
     };
   },
+
+  languageTexts: async (req: NextRequest) => {
+    const client = await getAdministrationServiceClient(req);
+    const languageTexts = client.languageTexts;
+    return {
+      get: async (baseCultureName = "ar", targetCultureName = "en") =>
```

Comment:
it would be great to have the base as English and the target as TR to let the whole team understand the page:  
```suggestion
      get: async (baseCultureName = "en", targetCultureName = "tr") =>
```

Reply:
done